app/assets/javascripts/dialog-polyfill.js

```diff
@@ -0,0 +1,533 @@
+(function() {
```

Comment:
Hi Matt. That's a JS file required to display the 'Dialog' (window that pops out when clicking 'Redeem your code' button and 'Delete your account') component of Material Design for browsers other than Chrome as I was using Firefox when I was restyling the CSS.

Only Chrome is compatible with the 'Dialog' component without the need for that JS file. Here's a link to the file:

https://github.com/GoogleChrome/dialog-polyfill

If it is affecting the app in any way we can remove it and just make do without the 'Dialog' window.